syntax = "proto3";

package dev.local;

option go_package = "./local";

message SampleObj{
  int32 id = 1;
  string name = 2;
  bool is_valid = 3;
  repeated string str_list = 4;
}

